docs/docs/configuration/overview.md

```diff
@@ -81,6 +81,7 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
 | `--auth-logging-format` | string | Template for authentication log lines | see [Logging Configuration](#logging-configuration) |
 | `--authenticated-emails-file` | string | authenticate against emails via file (one per line) | |
 | `--azure-tenant` | string | go to a tenant-specific or common (tenant-independent) endpoint. | `"common"` |
+| `--azure_graph_group_field` | string | Microsoft Graph API field to use for Azure group validation against `--allowed-group` ("id" or "displayName") | `"id"` |
```

Comment:
Shoudn't this be `--azure-graph-group-field`?

Reply:
Yes. Thanks